### ISSUE TITLE
ghc-8.0.2 issue

### ISSUE THREAD

**Issue by lancelet** [philopon/haddocset#40]
I'm seeing the following problem when trying to run haddocset on a project configured using stack (stackage lts-8.11; ghc-8.0.2):
```
$ stack exec -- haddocset -t target.docset create
[1/5] Create Directory.
[2/5] Writing plist.
[3/5] Migrate Database.
    Global package directory: /Users/jsm/.stack/programs/x86_64-osx/ghc-8.0.2/lib/ghc-8.0.2
    Global package count:     37
[4/5] Copy and populate Documents.
[5/5] Create index.
haddock: internal error: /Users/jsm/.stack/programs/x86_64-osx/ghc-8.0.2/lib/ghc-8.0.2/package.conf.d/package.cache: GHC.PackageDb.readPackageDb: inappropriate type (not enough bytes)
```

I built haddocset from the latest git master (43b770c) using stack.

**Comment by nrolland** [comment]
I have the same error. is some procedure necessary to fix it ?

**Comment by nrolland** [comment]
maybe tracking stackage would be a good idea

**Comment by philopon** [comment]
Hmm... It is not reproduced on my environment (both lts-8.11 and nightly-2017-06-11).
I don't know it's related or not, but I fixed compile error on ghc-8.0 at 13a90395b941f50241da5c49132b260d43e7dae6. Please test latest master.

**Comment by nrolland** [comment]
This should be it. Will test tomorrow.

**Comment by PierreR** [comment]
@philopon Would you mind release master on hackage ? That would make it work with the latest LTS-9 and the nix infrastructure.

**Comment by philopon** [comment]
I released v0.4.3.
enjoy:beer:
